src/routes/Profile.jsx

```diff
@@ -0,0 +1,78 @@
+// React Imports
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+// Utility Imports
+import { fetchProfileInfo } from '../model-helpers';
+// Component Inputs
+import ProfileInputField from '../components/Profile/ProfileInputField';
+
+/**
+ * Profile Page - Page that displays the user's profile card information and
+ * allow users to edit/update them on PASS
+ *
+ * @memberof Pages
+ * @name Profile
+ * @returns {React.JSX.Element} The Profile Page
+ */
+const Profile = () => {
+  const location = useLocation();
+  const { session } = useSession();
+
+  localStorage.setItem('restorePath', location.pathname);
+
+  const [profileName, setProfileName] = useState(null);
+  const [organization, setOrganization] = useState(null);
+
+  const fetchProfileData = async () => {
```

Comment:
naming?   The function does fetch the profile info, but ultimately sets the profile name and organization.

So, is the function doing too many things or just named a little funky?

Reply:
The function suppose to fetch profile info from the profile and set the local states. Perhaps I should rename this `setProfileData`?